firmware/config/boards/s105/board.mk

```diff
@@ -31,6 +31,9 @@ DDEFS += -DSTM32_USB_USE_OTG2=FALSE
 DDEFS += -DEFI_FILE_LOGGING=FALSE
 USE_FATFS = no
 
+# This board has different dividers on adc inputs
+DDEFS += -DEFI_NON_UNIFORM_ANALOG_DIVIDER=TRUE
```

Comment:
If we use `__attribute__((weak))`, we don't need any defines to select whether to use it or not.